### PR TITLE
Add drupal/tfa module to composer.json

### PR DESCRIPTION
**Pull request by sekinet813**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -110,8 +110,10 @@
     "drupal/openid_connect_daccount": "1.x-dev@dev",
     "drupal/password_policy": "^4",
     "drupal/queue_ui": "^3",
+    "drupal/real_aes": "^2.5",
```
Comment (naoi) [inline composer.json]: @sekinet813 Can we change the version to `^2` for the dev branch?  We can put the specific version to the official release.

File: composer.json [hunk 2]
```diff
@@ -110,8 +110,10 @@
     "drupal/openid_connect_daccount": "1.x-dev@dev",
     "drupal/password_policy": "^4",
     "drupal/queue_ui": "^3",
+    "drupal/real_aes": "^2.5", 
     "drupal/roleassign": "^2",
     "drupal/simple_oauth": "^5",
+    "drupal/tfa": "^2.0",
```
Comment (naoi) [inline composer.json]: @sekinet813 Can we change the version to `^2` for the dev branch?  We can put the specific version to the official release.

**Review by naoi — CHANGES_REQUESTED** [review]
@sekinet813 Thank you for the PR.  I posted my comments.

**Comment by KentaroHayashi-ipride** [comment]
@sekinet813 @naoi 
It looks good to me.

**Comment by sekinet813** [comment]
@naoi @KentaroHayashi-ipride Thank you for your review. I modified it as you suggested, could you please check it?

**Review by naoi — APPROVED** [review]

**Pull request merged**